### PR TITLE
fix: margins in shared post action button

### PR DESCRIPTION
**Pull request by el**
### Describe what this PR does
- Short and concise, bullet points can help
- Screenshots if applicable can also help

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-{number} #done

### CODE REVIEW

File: packages/shared/src/components/cards/SharePostCard.tsx [hunk 1]
```diff
@@ -74,7 +74,7 @@ export const SharePostCard = forwardRef(function SharePostCard(
           onShareClick={onShareClick}
           onMenuClick={(event) => onMenuClick?.(event, post)}
           onReadArticleClick={onReadArticleClick}
-          className="justify-between my-4 mx-4 laptop:mb-0"
+          className="justify-between my-4 mx-4 mt-0 laptop:mb-0"
```
Comment (sshanzel) [inline packages/shared/src/components/cards/SharePostCard.tsx:77]:
```suggestion
          className="justify-between mx-4 mb-4 laptop:mb-0"
```

**Review by sshanzel — APPROVED** [review]
Just had a suggestion but the fixes look good to me! 🚢

**Review by rebelchris — APPROVED** [review]
Besides Lee's comment, nothing from my side.

File: packages/shared/src/components/cards/SharePostCard.tsx (hunk 1 above)
Comment (el) [reply packages/shared/src/components/cards/SharePostCard.tsx]: Great suggestion

**Pull request merged**